src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksRetrieveAchUpdatesStep.java

```diff
@@ -44,197 +45,23 @@
-		} else {
-			achUpdateSummary.getRejectedVendors().add(summaryLine);
-		}
-	}
-
-	protected void updatePaymentWorksVendorStatus(String vendorRequestId, String requestStatus) {
-
-		// create update list of one
-		List<PaymentWorksUpdateVendorStatus> updateNewVendorStatusList = new ArrayList<PaymentWorksUpdateVendorStatus>();
-		PaymentWorksUpdateVendorStatus updateNewVendorStatus = new PaymentWorksUpdateVendorStatus();
-		updateNewVendorStatus.setId(new Integer(vendorRequestId));
-		updateNewVendorStatus.setStatus(new Integer(requestStatus));
-		updateNewVendorStatusList.add(updateNewVendorStatus);
-
-		paymentWorksWebService.updateVendorUpdatesStatusInPaymentWorks(updateNewVendorStatusList);
-	}
-
-	protected File writePaymentWorksAchUpdateSummaryReport(AchUpdateSummary achUpdateSummary) {
-
-		if (reportWriterService == null) {
-			throw new IllegalStateException("ReportWriterService not configured for PaymentWorks ACH Update service.");
-		} else {
-			reportWriterService.initialize();
-		}
-
-		List<AchUpdateSummaryLine> vendorsApproved = achUpdateSummary.getApprovedVendors();
-		List<AchUpdateSummaryLine> vendorsRejected = achUpdateSummary.getRejectedVendors();
-
-		writeAchUpdateEditSummaryDetailRecords("ACH Update Direct Edits", vendorsApproved);
-		writeAchUpdateRejectedSummaryDetailRecords("ACH Updates Rejected", vendorsRejected);
-
-		reportWriterService.writeStatisticLine("%d ach update direct edits", vendorsApproved.size());
-		reportWriterService.writeStatisticLine("%d ach updates rejected", vendorsRejected.size());
-		reportWriterService.writeStatisticLine("%d total ach updates processed",
-				vendorsRejected.size() + vendorsApproved.size());
-
-		return reportWriterService.getReportFile();
-	}
-
-	protected void writeAchUpdateEditSummaryDetailRecords(String subtitle, List<AchUpdateSummaryLine> records) {
-		String rowFormat = "%-15s %-30s %-20s";
-		String hdrRowFormat = "%-15s %-30s %-20s";
-		Object[] headerArgs = { "Vendor Req ID", "Vendor Name", "Vendor Number(s)" };
-
-		boolean firstPage = true;
-
-		for (AchUpdateSummaryLine vendorUpdateSummaryLine : records) {
-			if (reportWriterService.isNewPage() || firstPage) {
-				firstPage = false;
-				reportWriterService.setNewPage(false);
-				reportWriterService.writeSubTitle(subtitle);
-				reportWriterService.writeNewLines(1);
-				reportWriterService.writeFormattedMessageLine(hdrRowFormat, headerArgs);
-			}
-
-			reportWriterService.writeFormattedMessageLine(rowFormat, vendorUpdateSummaryLine.getVendorRequestId(),
-					vendorUpdateSummaryLine.getVendorName(), vendorUpdateSummaryLine.getVendorNumber());
+		List<PaymentWorksVendorUpdatesDTO> pendingACHUpdates = getPaymentWorksWebService().getPendingAchUpdatesFromPaymentWorks();
+		LOG.info("execute, number of ACH Updates retrieved: " + pendingACHUpdates.size());
+		boolean hasErrors = false;
+		routed = getPaymentWorksAchService().processACHUpdates(pendingACHUpdates, hasErrors);
+		if (hasErrors) {
```

Comment:
Thanks, yes I noticed this was part of the errors I was dealing with, I added a pojo to help with that